test/libsolidity/SMTCheckerTest.cpp

```diff
@@ -29,6 +29,10 @@ using namespace solidity::frontend::test;
 
 SMTCheckerTest::SMTCheckerTest(string const& _filename): SyntaxTest(_filename, EVMVersion{})
 {
+	auto contract = m_reader.stringSetting("SMTContract", "");
+	if (!contract.empty())
+		m_modelCheckerSettings.contracts.contracts[""] = {contract};
```

Comment:
Needed to add this option to SMTChecker isoltest otherwise the added tests would only report the assertion failure from L but we wanted to test C which is what caused the bug before.